### PR TITLE
Show learn more button catalog mobile

### PR DESCRIPTION
**Pull request by juanmanzojr**
On Mobile, if the Expanded Card cannot fit the width of the screen, the Quick View button turns back into its original Learn More button.

## Learn More button visible when Two curriculum Catalog Cards in a row
Screenshot 2023-11-27 at 9 51 10 AM

## Learn More button visible with one Catalog Card per row
Screenshot 2023-11-27 at 9 51 33 AM

## Quick View when the Expanded Card can fit the width of the screen
Screenshot 2023-11-27 at 9 50 35 AM

## Links

- jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?selectedIssue=ACQ-1027)

## Testing story
New UI tests within `dashboard/test/ui/features/acquisition_products/curriculum_catalog.feature`
## Follow-up work
Hide the Related Curricula section when there are two Catalog Cards per row in order to fit the Expanded Card.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss [hunk 1]
```diff
@@ -202,3 +203,31 @@ $container-width: 309px;
...
+  .arrowContainer {
+    display: none !important;
+  }
+
+  .centerExpandedCard {
+    display: none !important;
+  }
+
+  .learnMoreButton {
+    display: block !important;
```
Comment (bethanyaconnor) [inline apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss:228]: why are the `!important` tags necessary? What's overriding these styles?

Comment (juanmanzojr) [reply apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss:228]: For the learn more button, there is Button styling that makes the `display: inline block` which displays the Learn More button at all times. The `!important` overrides this display.

Comment (bethanyaconnor) [reply apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss:228]: Sorry for dragging this out but can you add a comment explaining that? `!important` tags are a huge anti-pattern so it's good to document why they're needed when they are.

**Review by bethanyaconnor — APPROVED** [review]
One request for a comment but otherwise looks good to me!

**Pull request merged**